src/components/FavoriteCities/FavoriteCityForecast/FavoriteCityForecast.jsx

```diff
@@ -1,23 +1,49 @@
+// @flow
+import React, { useState, useEffect } from 'react';
 import classes from './FavoriteCityForecast.module.scss';
+import ApiService from '../../../services/ForecastApiService';
+import { ReactComponent as IconClose } from '../../../assets/img/svg/close-icon.svg';
+import { FORECAST_SYMBOL_LINK, FORECAST_SYMBOL_EXT } from '../../../utils/constants';
+import type { FavoriteCityForecastPropsType } from './FavoriteCityForecastPropsType';
+
+function FavoriteCityForecast({ location }: FavoriteCityForecastPropsType): React$Node {
+  const [forecast, setForecast] = useState({});
+
+  useEffect(() => {
+    const setForecastValue = async (): Promise<void> => {
+      let currentForecast = {};
```

Comment:
I suppose it'd be better to move everything here to redux (action and state): the only code that should be left here is something like: 
```
useEffect(() => {
  props.getForecast();
}, [props.getForecast]);
```

note that `const [forecast, setForecast] = useState({});` is gone too. 

setForecast is moving to reducers, forecast is getting from redux state

Walk through all of your components and make sure there is no service imports. Unbind them all.

Reply:
I will connect redux in future PR, because this PR has a lot of changed files.